src/components/ProfileButton.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import { openSans } from '@/styles/fonts';
+import Link from 'next/link';
+import COLORS from '@/styles/colors';
+
+const ProfileButton = styled(Link)`
+  ${openSans.style}
+
+  display: flex;
+  align-items: center;
+  gap: 1.125rem;
```

Comment:
violating the figma but...thoughts on reducing to `gap: 0.7rem;`? (@davidqing6432)

Reply:
only bc the profile icon and the name feel like 2 different entities rn

Reply:
or `gap: 0.75rem` 🤔

Reply:
sure yeah its not super super significant